### DISCUSSION TITLE
How do you generate defaultJasprOptions?

### DISCUSSION THREAD

**Discussion by MarkOSullivan94** [schultek/jaspr#343] [Q&A]
My project doesn't have `defaultJasprOptions` and I'm curious how to generate it

**Comment by schultek** [comment]
Whats your setup? It should be generated automatically.

**Reply by MarkOSullivan94** [reply]
I'm using client mode though I believe I created this project before `defaultJasprOptions` existed

Is it generated when I run `jaspr serve`?

**Reply by schultek** [reply]
Client mode doesn't need it. You can just ignore it.
